### DISCUSSION TITLE
Google/Numpy style docstrings?

### DISCUSSION THREAD

**Discussion by rmorshea** [sphinx-extensions2/sphinx-autodoc2#15] [Q&A] [answered]
Is sphinx-autodoc2 compatible with Google/Numpy style docstrings?

**Comment by chrisjsewell** [comment, marked as answer]
Heya, see the discussion at the bottom of https://github.com/executablebooks/MyST-Parser/issues/228 (and feel free to comment)

Basically, no. It at the moment, it's a work in progress
